packages/gestalt/src/TextField.js

```diff
@@ -8,7 +8,12 @@ import FormErrorMessage from './FormErrorMessage.js';
 import styles from './TextField.css';
 
 type Props = {|
-  autoComplete?: 'current-password' | 'on' | 'off' | 'username',
+  autoComplete?:
```

Comment:
can we update the TextField.doc.js file as well for this new option and add a CHANGELOG.md entry for this please